core/src/nucleus/ribosome/callback/validate_entry.rs

```diff
@@ -113,9 +113,14 @@ fn validate_link_entry(
         direction: link_definition_path.direction,
         validation_data,
     };
+
     let call = ZomeFnCall::new(
         &link_definition_path.zome_name,
-        None,
+        make_internal_capability_request(
```

Comment:
Could this stay `None` if `run_validation_callback` below would do a `DirectCall`, which it should?

Reply:
Hmm.  Maybe this makes sense.  I'll look at that.